invenio_app_ils/config.py

```diff
@@ -572,7 +572,7 @@ def _(x):
             ),
             available_items=dict(
                 terms=dict(
-                    field="circulation.loanable_items",
+                    field="circulation.available_items",
```

Comment:
Does this refer to items that can circulate? If so imho introducing the yet another term `available` will cause confusion in the future.

Reply:
I have changed it for items_available_for_loan which is more explicit